torchsynth/module.py

```diff
@@ -571,19 +533,180 @@ def __init__(
         **kwargs: Dict[str, T],
     ):
         super().__init__(synthglobals, **kwargs)
-        self.register_buffer(
-            "phase", self.get_parameter("initial_phase").detach().clone()
-        )
+
         # Create and save a buffer of sample points
         self.register_buffer("range", torch.arange(self.buffer_size))
 
+    def create_arg(self) -> Tuple[T, T]:
+        """
+        Creates the arg signal (phase over time)
+        """
+        frequency = self.p("frequency").unsqueeze(1)
+        rads = 2 * torch.pi * frequency / self.sample_rate
+        arg = self.range.expand(self.batch_size, -1) * rads
+        arg += self.p("initial_phase").unsqueeze(1)
+        return arg, frequency
+
+    def _forward(self) -> Signal:
+        """
+        Must be implemented in deriving classes
+        """
+        raise NotImplementedError
+
+
+class SineLFO(AbstractLFO):
+    """
+    A sine wave low-frequency oscillator
+    """
+
+    def _forward(self) -> Signal:
+        """
+        Creates a batch size number of LFO modulation signals with values [0,1]
+        """
+        arg, _ = self.create_arg()
+        return ((1.0 + torch.cos(arg)) / 2.0).as_subclass(Signal)
+
+
+class SquareSawLFO(AbstractLFO):
```

Comment:
We could get away with a much simpler square/saw wave oscillator for LFO's because we're not worried about aliasing.

Reply:
Could have a simple sine, square, saw, and maybe triangle. Mickey mouse though, nice and easy.